### ISSUE TITLE
Player height changes when entering/exiting doors

### ISSUE THREAD

**Issue by benjamincheng10** [Okabintaro/SubmersedVR#47]
hi, I'm experiencing a bug where my player would become twice as tall sometimes when entering or exiting any doors in bases or vehicles. so sometimes i'll get into my vehicles and my head would be sticking out of the roof, or my head goes into the ceiling in buildings. it usually resets after I enter/exit any door again. Might have something to do with the way the game registers player height when playing seated?

**Comment by benjamincheng10** [comment]
yeah I'm using OVR Advanced settings, not space drag tho. i'll try disabling OVR and see if it still happens, thanks!

**Comment by NeonPixels** [comment]
It might be unrelated, but on my first test of Subnautica VR (without mods) I had a similar issue, after starting a new game (on creative) the player was very tall when inside the pod, but after exiting through a hatch, the height reset to the correct value.
Maybe it is an underlying Subnautica issue?
